### DISCUSSION TITLE
Import *.prma via batch

### DISCUSSION THREAD

**Discussion by BLaX50** [1Remote/1Remote#288] [Q&A]
Hi and thank you for your work.
I wanted to know if we can import *.prma to PRemoteM via some batch to use with GPO ?
Thanks.

**Comment by VShawn** [comment]
Uh.... I didnt get why we need `Import *.prma via batch`

do you want to share some server in your team by run the import by the GPO?

**Reply by BLaX50** [reply]
You're right.
I already deployed PRemoteM via GPO/Powershell but I need to "auto import via *.prma" servers on the app and not manually on each computer.
But it's just a question :)

**Reply by VShawn** [reply]
then I'm working on #282, which can help. 


![image](https://user-images.githubusercontent.com/10143738/190358768-94dcf575-3b73-4ac4-9691-1f203ff08c39.png)

I desigen different data sources for team use. 

Personal servers saved in local SQLite database, team shared serves saved online MySQL database.

Still WIP. The workload exceeds my expectations

**Reply by VShawn** [reply]
For a temporary solution, maybe you can share database by GPO instead?

**Reply by BLaX50** [reply]
Yes I will do that thank you :)

**Comment by andy1463** [comment]
Hello,
I wanted to ask if there is a plan for importing prma from batch.
We have different employees which need access to different hosts. So import prma from batch would be perfect for deploy.
Thanks a lot

**Reply by VShawn** [reply]
No we dont going to make this plan for now. but here's the feature you may interesting: https://1remote.github.io/usage/team/team-sharing/

**Reply by andy1463** [reply]
Thank you for the response.
I wanted to try the team sharing feature but I don´t have the possibility to add a mysql database. I only can choose a local sqlite database.
Can you tell me how to connect to a mysql database?
Thanks a lot

**Reply by VShawn** [reply]
well you can connect mysql with our preview version, BTW it is easy for you to deploy mysql with docker

**Reply by andy1463** [reply]
Oh sorry, I tried with the wrong version. Is there a plan when the new version will be released?

**Reply by VShawn** [reply]
Currently, the release of the new version still requires some documentation work, so although its development has been completed, it is still in preview status. However, I believe that the new version will be fully released within the next one or two months

**Reply by andy1463** [reply]
That is great news. Thank you very much for your work.
